ocfweb/docs/docs/staff/getinvolved.md

```diff
@@ -0,0 +1,227 @@
+  approve new account requests. Written in Python.
+   * See https://ircbot.ocf.berkeley.edu/ for a listing of IRC bot functionality
+
+#### TODOs
+
+If you’re feeling particularly adventurous, you can [search for the string
+“TODO” across our entire codebase][sourcegraph-todo].
+
+#### Help requests
+
+During the semester, our [[help@ocf.berkeley.edu mailing list|doc contact]]
+gets around 14 emails per week from the UC Berkeley community. These emails get
+responded to by volunteers like you. Most requests are about requesting virtual
+hosting and helping users debug or fix their websites. If you’re interested in
+seeing how this process works, ask a staffer to add you to the RT mailing list.
+Since this is high-volume, we recommend filtering these to a different folder
+so you don’t get notified all the time.
+
+Joining this list doesn’t obligate you to respond to help requests. It’s
+perfectly acceptable to only read tickets as a way to learn more about what we
+do.
+
+If you’re on staff, you can view the ticket archive at rt.ocf.berkeley.edu.
+
+#### Your idea here
+
+As a volunteer organization, the OCF’s direction is driven by the interests of
+our members. If you have a new idea for the OCF, we want to help you build it.
+You can use our chat channels or talk to a veteran staffer in person to get a
+starting point and roadmap.
+
+### Staff Hours
+
+During the week, veteran staffers host “staff hours”
+([ocf.io/staffhours][staffhours]), where we provide support to users of our
+services. As a newbie, attending staff hours can serve multiple purposes:
+
+   * Learn by shadowing veteran staffers as they help people out
+   * Talk about your interests so we can help you find stuff to work on
+   * Get one-on-one mentoring so you can learn about the vast complex world of
+     OCF infrastructure
+   * Get context and more information on a particular area you’re interested
+     in contributing in
+
+### Slack or IRC
+
+OCF staff often communicate on Slack or IRC. For instructions on how to join
+Slack, see [ocf.io/slack][slack]. Some important channels:
+
+   * #rebuild - technical discussion
+   * #administrivia - non-technical discussion
+   * #ocf for off-topic, non-OCF related discussion
+
+These channels are typically very active, and you shouldn’t feel the need to
+read every line to stay updated. You also aren’t required to understand all the
+technical jargon that’s used in #rebuild. Feel free to introduce yourself in
+any of these channels if you’re new, and don’t be afraid to ask questions!
+
+### Hang out in the lab
+
+OCF staff members usually like to socialize in the lab. We’re a friendly bunch,
+so feel free to talk to us! You can also come into the lab after-hours-- just
+say you’re interested in joining staff and we’ll let you in.
+
+## FAQ
+
+**I wasn’t able to make the first/second/nth meeting of the semester, can I
+still join?**
+
+Yes! You can join at any time. If you’re worried about what you’ve missed, we
```

Comment:
> worried about what you've missed
perhaps also mention they should feel free/comfortable to ask veteran staff to bring them up to speed